doc/en/components/charts/types/bubble-chart.md

```diff
@@ -48,7 +49,7 @@ In this example, the Bubble Chart is comparing the total births and deaths betwe
 
 ## $PlatformShort$ Bubble Chart Styling
 
-The Bubble Chart contains many of the characteristics of the Scatter Point Chart but with the option to have various radius scale sizes and fill scale colors.
+The Bubble Chart contains many of the characteristics of the [Scatter Marker Chart](scatter-chart.md) but with the option to have various radius scale sizes and fill scale colors.
```

Comment:
append "#$PlatformShort$-scatter-marker-chart" after ".md" to make this a link to a section of that topic

Reply:
pending